### PR TITLE
Updates to model options page 

### PR DESCRIPTION
**Pull request by jeffeaton**
This PR updates the model options page (#161) per discussion w/ UNAIDS. 

Currently completed (less any errors in my JSON):

* Calendar quarter for current estimates to Dec 2021.
* Move short term projection calendar quarter to General block, remove 'Output options' block.
* Add options for number or months reporting in ANC testing data block.
  * This involves a re-organisation of the ANC testing data block to be four rows.
* Add an endpoint `<+calendar_quarter_t1_default+>` for most recent survey midpoint as default for `calendar_quarter_t1`.
* Add fields for option to use Kish design effect in Advanced block.

Two new strings are added to the English dictionary. French translations need to be added or edited:

* `"OPTIONS_ANC_MONTHS_LABEL": "Number months included",`
* `"OPTIONS_ANC_MONTHS_HELP": "Number of months included in ANC reporting in this year.",`
* `"OPTIONS_ADVANCED_USE_KISH_LABEL": "Use Kish ESS",`
* `"OPTIONS_ADVANCED_USE_KISH_HELP": "Use Kish Effective Sample Size (ESS) for survey data likelihood"" `
* `"OPTIONS_ADVANCED_DEFF_PREVALENCE_LABEL": "Survey design effect - HIV prevalence", `
* `"OPTIONS_ADVANCED_DEFF_ART_COVERAGE_LABEL": "Survey design effect - ART coverage",`
* `"OPTIONS_ADVANCED_DEFF_PROPORTION_RECENT_LABEL": "Survey design effect - Proportion recently infected",`
* `"OPTIONS_ADVANCED_ESS_SCALING_LABEL": "ESS Scaling Factor"`

It would be helpful to see this on staging.

### CODE REVIEW

**Comment by jeffeaton** [comment]
This is ready for review. Working on corresponding hintr PR to supply the endpoint `<+calendar_quarter_t1_default+>`.

File: inst/metadata/advanced_run_options.json [hunk 1]
```diff
@@ -199,8 +169,26 @@
     {
       "label": "t_(OPTIONS_ADVANCED_DEFF_PROPORTION_RECENT_LABEL)",
       "controls": [
+          "name": "use_kish_recent",
```
Comment (r-ash) [inline inst/metadata/advanced_run_options.json:172]: Missing an `{` here.

You can always paste this json into a validator. Either something like https://jsonlint.com/ or in R itself `jsonlite::read_json("path/to/file")` note that things which use the templating will fail though as template is not valid json. Using an online tool should help distinguish between actual errors and just ones from templating

File: inst/metadata/anc_run_options.json [hunk 2]
```diff
@@ -14,6 +14,69 @@
             <+anc_prevalence_year1_options+>,
           "required": false
         },
+        {
+          "name": "anc_prevalence_year1_num_months",
+          "label": "t_(OPTIONS_ANC_MONTHS_LABEL)",
+          "type": "select",
+          "value": 12
```
Comment (r-ash) [inline inst/metadata/anc_run_options.json:21]: Missing a `,` here

File: inst/metadata/anc_run_options.json [hunk 3]
```diff
@@ -22,11 +85,69 @@
           "options":
             <+anc_prevalence_year2_options+>,
           "required": false
+        },
+        {
+          "name": "anc_prevalence_year2_num_months",
+          "label": "t_(OPTIONS_ANC_MONTHS_LABEL)",
+          "type": "select",
+          "value": 12
```
Comment (r-ash) [inline inst/metadata/anc_run_options.json:93]: Another `,`

File: inst/metadata/anc_run_options.json [hunk 4]
```diff
@@ -37,6 +158,69 @@
             <+anc_art_coverage_year1_options+>,
           "required": false
         },
+        {
+          "name": "anc_art_coverage_year1_num_months",
+          "label": "t_(OPTIONS_ANC_MONTHS_LABEL)",
+          "type": "select",
+          "value": 12
```
Comment (r-ash) [inline inst/metadata/anc_run_options.json:165]: ,

File: inst/metadata/anc_run_options.json [hunk 5]
```diff
@@ -45,6 +229,64 @@
           "options":
             <+anc_art_coverage_year2_options+>,
           "required": false
+        },
+        {
+          "name": "anc_art_coverage_year2_num_months",
+          "label": "t_(OPTIONS_ANC_MONTHS_LABEL)",
+          "type": "select",
+          "value": 12
```
Comment (r-ash) [inline inst/metadata/anc_run_options.json:237]: , :fearful:

File: inst/metadata/general_run_options.json [hunk 6]
```diff
@@ -36,10 +36,72 @@
...
+                     {
+              "id": "CY2022Q4",
+              "label": "t_(MONTH_DECEMBER) 2022"
+            }
+          ]
+        }
       ]
     }
-  ]
+  }
```
Comment (r-ash) [inline inst/metadata/general_run_options.json:106]: This an intended switch?

File: inst/traduire/fr-translation.json [hunk 7]
```diff
@@ -84,7 +84,7 @@
     "OPTIONS_ADVANCED_DEFF_ART_COVERAGE_HELP": "DEFF approximate pour ajuster le taille effective de l'échantillon pour l'enquête survey sur le couverage d'ART",
     "OPTIONS_ADVANCED_DEFF_PROPORTION_RECENT_LABEL": "District approximate enquête design éffet - Proportion de personnes récemment infectées",
     "OPTIONS_ADVANCED_DEFF_PROPORTION_RECENT_HELP": "DEFF approximate pour ajuster le taille effective de l'échantillon pour la proportion récemment infectée parmi les séropositifs",
-    "OPTIONS_OUTPUT_PROJECTION_QUARTER_LABEL": "Trimestre de projection:",
+    "OPTIONS_OUTPUT_PROJECTION_QUARTER_LABEL": "Trimestre de projection pour générer une projection à court terme",
     "OPTIONS_OUTPUT_PROJECTION_QUARTER_HELP": "Trimestre calendaire pour générer une projection à court terme. Doit être après le trimestre calendaire pour générer des estimations",
```
Comment (r-ash) [inline inst/traduire/fr-translation.json:88]: I wonder if @richfitz can give us some translations? Alternatively put through google translate and then ask UNAIDS to get them checked

**Review by r-ash — CHANGES_REQUESTED** [review]
A few json issues, does emacs not have syntax highlighting for JSON?

File: inst/metadata/advanced_run_options.json (hunk 1 above)
Comment (jeffeaton) [reply inst/metadata/advanced_run_options.json:173]: Thanks, fixed in 73e91ce.

File: inst/metadata/anc_run_options.json (hunk 2 above)
Comment (jeffeaton) [reply inst/metadata/anc_run_options.json]: Thanks, fixed in 73e91ce.

File: inst/metadata/anc_run_options.json (hunk 3 above)
Comment (jeffeaton) [reply inst/metadata/anc_run_options.json]: Thanks, fixed in 73e91ce.

File: inst/metadata/anc_run_options.json (hunk 4 above)
Comment (jeffeaton) [reply inst/metadata/anc_run_options.json]: Thanks, fixed in 73e91ce.

File: inst/metadata/anc_run_options.json (hunk 5 above)
Comment (jeffeaton) [reply inst/metadata/anc_run_options.json]: Thanks, fixed in 73e91ce.

File: inst/metadata/general_run_options.json (hunk 6 above)
Comment (jeffeaton) [reply inst/metadata/general_run_options.json]: Thanks, fixed in 73e91ce.

File: inst/traduire/fr-translation.json (hunk 7 above)
Comment (jeffeaton) [reply inst/traduire/fr-translation.json:88]: Merci!

**Comment by jeffeaton** [comment]
I have added one more commit on this updating the Calibration options help text on model options page, addresses #155.

**Review by r-ash — APPROVED** [review]

**Pull request merged**